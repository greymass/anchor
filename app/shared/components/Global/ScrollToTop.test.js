import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ScrollToTop from './ScrollToTop';

describe('ScrollToTop', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ScrollToTop />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});

