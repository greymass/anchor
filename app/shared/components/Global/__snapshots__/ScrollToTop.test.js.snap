// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop renders correctly 1`] = `
<Route>
  <Component />
</Route>
`;
